plugin_sets:
  default:
    # StateGeneratorPlugin
    reachability_state_generator: none

    # StepPlanMsgPlugin
    step_plan_msg_plugin: none

    # ReachabilityPlugins
    reachability_polygon: none

    # StepCostEstimatorPlugins
    const_step_cost_estimator: none
    euclidean_step_cost_estimator: none
    ground_contact_step_cost_estimator: none

    # HeuristicPlugins
    euclidean_heuristic: none
    dynamics_heuristic: none
    step_cost_heuristic: none

    # PostProcessPlugins
    step_dynamics_post_process: none

    # CollisionCheckPlugins
    2_upper_body_grid_map_model:
      import: upper_body_grid_map_model
      params:
        collision_check_flag: 2 # Upper Body
        grid_map_topic: /body_level_grid_map
    3_foot_grid_map_model:
      import: foot_grid_map_model
      params:
        collision_check_flag: 1 # Foot
        grid_map_topic: /ground_level_grid_map

    # TerrainModelPlugins
    1_terrain_model:
      import: terrain_model
      params:
        collision_check_flag: 4 # Foot Contact Support
        terrain_model_topic: /terrain_model
